src/Resources/config/doctrine/model/GiftCardConfiguration.orm.xml

```diff
@@ -0,0 +1,29 @@
+<doctrine-mapping xmlns:gedmo="http://gediminasm.org/schemas/orm/doctrine-extensions-mapping"
+                  xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping">
+
+    <mapped-superclass name="Setono\SyliusGiftCardPlugin\Model\GiftCardConfiguration" table="setono_sylius_gift_card__configuration">
+        <id name="id" column="id" type="integer">
+            <generator strategy="AUTO"/>
+        </id>
+
+        <field name="code" unique="true"/>
+        <field name="enabled" type="boolean"/>
+
+        <field name="createdAt" column="created_at" type="datetime">
+            <gedmo:timestampable on="create"/>
+        </field>
+
+        <field name="updatedAt" column="updated_at" type="datetime" nullable="true">
+            <gedmo:timestampable on="update"/>
+        </field>
+
```

Comment:
You are missing a relation to a locale, aren't you?

Reply:
Yes indeed, will be fixed.